cmake/NeuronTestHelper.cmake

```diff
@@ -143,6 +143,8 @@ function(nrn_add_test_group)
     # Add a rule to build the modfiles for this test group. Multiple groups may ask for exactly the
     # same thing (testcorenrn), so it's worth deduplicating.
     set(hash_components ${NRN_ADD_TEST_GROUP_NRNIVMODL_ARGS})
+    # Escape special characters (problematic with Windows paths when calling nrnivmodl)
+    string(REGEX REPLACE "([][+.*()^])" "\\\\\\1" NRN_RUN_FROM_BUILD_DIR_ENV "${NRN_RUN_FROM_BUILD_DIR_ENV}")
```

Comment:
@jorblancoa : could you copy here the problematic path here in this comment? (as an example / as a future reference)